#include "meminfo.pb.h"

const char* c_mem_total = "MemTotal:";
const char* c_mem_free = "MemFree:";
const char* c_mem_buffers = "Buffer:";
const char* c_mem_cached = "Cached:";
const char* c_swap_cached = "SwapCached:";
const char* c_hight_mem_total = "HighTotal:";
const char* c_hight_mem_free = "HighFree:";
const char* c_low_mem_total = "LowTotal:";
const char* c_low_mem_free = "LowFree:";
const char* c_swap_total = "SwapTotal:";
const char* c_swap_free = "SwapFree:";
